### ISSUE TITLE
Suport for Dacia Spring

### ISSUE THREAD

**Issue by iboboc** [fesch/CanZE#742]
Dacia Spring shares the same structure of Zoe, so with some changes, it can be supported as well, knowing that already are thousands of owners.

I intend to make those changes, if there are developers that wants to help, let me know.
In the same time I need to know if there is interest from the owners of the code, willingness to merge those changes and support Dacia Spring as well.

**Comment by yoh-there** [comment]
As far as I am concerned (I am not the only one here in town!) there is absolutely willingness and enthusiasm to include the Dacia Spring, or any comparable car really. Really the only thing is the original authors do not own such cars and we had a hard time getting ZE50 to work properly. Basically it was only resolved when committed developer-owner were on board. Seems like you are in that position and that is wonderful!

Personally I have no problem merging in PR's especially into beta, and to put some announcements out, as long as it doesn't break current functionality of course.

Calling on @fesch re. his opinion (and the others chime in too of course!)

**Comment by Leobaillard** [comment]
I'm with @yoh-there on this one. We suspected that the architecture would be similar, allowing for a common base.

I only have a ZE50 but I'm willing to assist your efforts in any way I can. Feel free to ask for help if need be. Maybe we can share a more direct communication channel than the Github issue if you need extended help.

**Comment by fesch** [comment]
Yes, as long as the base of CanZE stays, the same, merging will be fine I think ...

**Comment by iboboc** [comment]
Thanks you, I'll prepare the code changes for review. I have tested myself, all features are fully working, the exact functionalities for Zoe now it works for Spring as well.

All the changes are made, has been done by keeping the original compatibility, nothing removed, so there should be zero impact. Since I've deviated a lot from original (for testing purposes, I have to cleanup the code and manually create the patch, it will take me few days.

Thanks.
